api/src/main/resources/notebooks.yaml

```diff
@@ -562,13 +674,68 @@ definitions:
       labels:
         type: object
         description: The labels to be placed on the cluster. Of type Map[String,String]
+      errors:
+        type: array
+        description: The list of errors that were encountered on cluster create. Each error consists of the error message, code and timestamp
+        items:
+          $ref: "#/definitions/ClusterError"
+      instances:
+        description: Array of instances belonging to this cluster
+        type: array
+        items:
+          $ref: '#/definitions/Instance'
+
+  InstanceKey:
+    description: ''
+    required:
+      - project
+      - zone
+      - name
+    properties:
+      project:
+        type: string
+        description: The Google Project the instance belongs to
+      zone:
+        type: string
+        description: The Google zone the instance belongs to
+      name:
+        type: string
+        description: The name of the instance
+
+  Instance:
+    description: ''
+    required:
+      - key
+      - googleId
+      - status
+    properties:
+      key:
+        description: Unique identifier of (project, zone, name) for this instance
+        $ref: '#/definitions/InstanceKey'
+      googleId:
+        type: string
+        description: Google's unique id for this instance
+      status:
+        $ref: "#/definitions/InstanceStatus"
```

Comment:
This file is just a copy-paste from Leo, we should fix it at the source if we want consistency - don't feel strongly enough to send that change in isolation, but if I push another fix upstream I'll do this.

FYI: part of these inconsistencies may have been my fault, since I pushed some validation fixes upstream: https://github.com/DataBiosphere/leonardo/pull/373

Reply:
👍